code/modules/mob/living/simple_animal/abnormality/he/laetitia.dm

```diff
@@ -168,6 +179,19 @@
 		P.TriggerPrank()
 	return
 
+/mob/living/simple_animal/hostile/abnormality/laetitia/BreachEffect(mob/living/carbon/human/user, breach_type)
+	if(breach_type == BREACH_MINING)
+		breaching = TRUE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/laetitia/proc/SummonAdds()//Mining breach summon
+	summon_cooldown = world.time + summon_cooldown_time
+	if(LAZYLEN(summon_list) > 9)//this list is not subtracted when minions are killed. Limited to 10 per breach
```

Comment:
"this list is not subtracted when minions are killed. Limited to 10 per breach" Whats the point of having a list then???
Replace this with a number, this is not only wastefull and can confuse coders but also makes the mobs hard-delete themselfes due to them having a reference being kept